<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CNBGeneratorSettingsEditorWidget</class>
 <widget class="QFrame" name="CNBGeneratorSettingsEditorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>259</width>
    <height>145</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CNBGeneratorSettingsEditorWidget</string>
  </property>
  <layout class="QFormLayout" name="mainLayout">
   <property name="horizontalSpacing">
    <number>5</number>
   </property>
   <property name="verticalSpacing">
    <number>5</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="lblGPUCoreRating">
     <property name="text">
      <string>GPU core rating:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="sbGPUCoreRating">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1000000</number>
     </property>
     <property name="singleStep">
      <number>1000</number>
     </property>
     <property name="value">
      <number>1000</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblLimitationsDivider">
     <property name="text">
      <string>Limitations divider:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="sbLimitationsDivider">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1000000</number>
     </property>
     <property name="singleStep">
      <number>32</number>
     </property>
     <property name="value">
      <number>128</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblSmallTPBPenaltyWeight">
     <property name="text">
      <string>Small TPB penalty weight:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="sbSmallTPBPenaltyWeight">
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="maximum">
      <double>10000000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.000010000000000</double>
     </property>
     <property name="value">
      <double>0.700000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblLargeTPBPenaltyWeight">
     <property name="text">
      <string>Large TPB penalty weight:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDoubleSpinBox" name="sbLargeTPBPenaltyWeight">
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="maximum">
      <double>10000000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.000010000000000</double>
     </property>
     <property name="value">
      <double>0.050000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblMultiplicativeLengthScaleFactor">
     <property name="text">
      <string>Multiplicative length scale factor:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDoubleSpinBox" name="sbMultiplicativeLengthScaleFactor">
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="maximum">
      <double>10000000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.000010000000000</double>
     </property>
     <property name="value">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lblAdditiveLengthScaleFactor">
     <property name="text">
      <string>Additive scale length factor:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QDoubleSpinBox" name="sbAdditiveLengthScaleFactor">
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="maximum">
      <double>10000000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.000010000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
